examples/notebooks/Watch_Warning_and_Advisory_Plotting.ipynb

```diff
@@ -0,0 +1,721 @@
t+/iJ95+DBg+pbtmzRev36NbW6uprAZrPrra2tqydPnlwUEhJS2dF1VZYvtXEo27cV52mPHj16am9v32AswOQdwD/zti94jWbB4XANxpivTUeMJ9+ingCt25h+dL5YCHxPI2t7yxcKhbiLFy+yAABOnz7N2bFjx3sSidRxlftKsFgs8YkTJzIbH//R604ikcDBwUF4584dhuJxsVgMDx8+VKFSqdKkpCSGoqE1Pj6eIZFIwMfH55tOxpYvX55XXl5e8C3LRCjP9zSydkT50dHRr/X19RtMzBwdHZUyVP4oHDx4kCUUCgkAANHR0Rx3d/evuhASEhJSYWNjk66vr98mQxbix+J7GlnbW/6SJUu0VqxY0cnV1VWwcOHCD2ZmZqLKykpCamqqyqFDh7gPHz5UuXHjxquOrG9b8PLyEhw9elQjKSlJpWfPnkLs+I0bN5hUKlWalpbGaHxPWloa08XFRfitjKwAAB4eHlXx8fHp/zaZh1CO72lk/Zblf48xycDAoF5fX1908+ZNJgDIJ8a5ubnEN2/eUKlUqjQhIYGpODFOSEhgAgAEBAQIAQB27Njx9lvVF8PPz0+wdu1aiIuLY5iZmZVix69fv86gUqnSFy9e0CoqKvBqampS7NytW7cYFhYW1d/KyAoAoK+vXx8fH59uZWUlav1qBKJlvqeRtT3lr1y5UnPx4sV6gwcPLg4PD89nMBjSV69eUS5duqQWHx+v+j0NrWFhYUV9+vSpaP3K9qGioiL966+/OFu2bGlg9Ny/fz9HRUVFWlVV1eFKU3x8fLqhoeE3cyTpiPHkW/E9xq2O5D+ZOuDQoUMsoVBI8PHxqSgtLSXGxsaqKXNfTU0N7mvXrSVIJJIsICCgqvG/71knZenataswJyeHkp2dLbcK37p1iy4UCgnDhg0rvnXrFlPx+qSkJCYOh4MePXq0q0O39TezsbERde3aFU0CEV8FFxeX6sb9l81mS1u/88fh0KFDHDU1NYm9vX3V6dOnOWKx+KuWp6urKw4ICKii0Wiy1q9uH1KpFGpra7+rnEf8WJw7d465YsWKTuPGjSu8devWy2nTppUEBgYKhw4dWhEREZH75s2bJyEhIaWtP+nr0aNHDwHAR+OF4vE7d+4wQkNDi4uLi0mPHz+mYMcfPXpEKSkpIXp5ebVrIVMsFkN9vfK2JjabLf03yjwEQpFvOSYp4urqKnj48KGKSCSSj1FXr15lMBgMSZ8+fcqSk5Mb9P+UlBSGmpqaxMXFpQYAwNnZuVZx4vwt8Pb2rqZSqdLk5OQGOn5qaipz4MCBpSQSSZaQkCCvd2VlJf758+d0d3f3dun+bR3LaTSaLCAgoEpXV/frKjQIxA9MZGSkVrdu3cqPHz/+NjQ0tKJ///6CmTNnFsfHx2dFRkZ+t+hfAAATE5P6b2Hz6NmzZ1lsbCxHKv1HTREKhbjLly+r9+rVq6wjy8JsFAEBAVUmJibf1JmkvePJl9JWu8z3GLc6kv+qoZWjqqoqOXLkSDaVSpUePHiwSUh+eHi4Lg6Hc05LS6N6enqa0el0x759+xoDAAgEAvwvv/zC4/F4diQSyYnH49nNnTtXWyL5Z/G1uroa99NPP+mZmZnZ0Ol0Rw0NDb6/v7/pw4cPqd/wVaG0tBQ/evRofU1NTXsymexkaGhou2zZMk1FAQLwceLVvXt3EyaT6UClUp34fL5lbGysauPnRUVFsYyMjGzIZLKTqampzYEDB9SVqYe/v78A4GMnxo5du3aNYWxsXDto0KDyp0+f0gUCgfx7vHXrFtPMzKyGy+VKlG3LiIgIDg6Hc7506RIjMDDQmMlkOjg6OloBAPB4PLsBAwYYRUVFsYyNjW1oNJqjra2t1ZUrVxoIkkGDBhnyeDw77O+MjAwyDodz/vPPPzWmT5+uy+Vy7ZlMpoO/v79pVlZWA1digUCAHzFihL66urqDioqKY/fu3U3i4uJUcDicc0RERLNpHxCIlpBKpbBs2TJNQ0NDWxKJ5MTlcu1Hjx6tX1pa2kB2K9vPU1NTac7OzhYUCsVJU1PTfvbs2ToymfJzxTdv3pBu376t2q9fv9IxY8YUl5SUEE+ePNlkoer58+dkX19fUxqN5shisfjjxo3TUxzMAQB8fHxMbWxsrBrf+/btWxKRSHRevny5JsA//TojI4OMXbNz5062lZWVNZ1Od2QymQ7m5ubWf/75p4bic7Zv3862sLCwplAoTiwWix8UFGT09u3bBn0WkwubN2/mYHLt4MGD6iwWi//TTz/pNa4bVpdvLccR349169Zpq6uriz83yVBVVZU2DkebMWOGrrW1tRWTyXRgsVh8Nzc384SEBBVlytu/f786n8+3pNFojkwm0yEwMND41atX5JbuMTExqefxeHWpqalyY0ZBQQEhMzOTNnLkyBJdXd26+Ph4+Tns/9i4HBUVxXJzczNnsVh8Op3uaGVlZb1169YmYxYOh3P+9ddfeQsWLNDm8Xh2FArF+e7duzRMX3ry5AnF19fXlE6nO+rq6trNmjVLR1EvOn/+PBOHwzmfP39eXpcuXbpYODs7W5w5c4ZpbW1tRaPRHM3MzGwOHjyo3rj8Xbt2sY2MjGwoFIqTubm59eHDh9W6dOligYWWIRDfgubGpLbqf9hYTKPRHA0MDGzXrVvHba1cb29vYW1tLT4lJYWOHUtKSmI6OTkJvb29BXfv3m1gzLx79y7T2dlZgHmtN+4rWH88fPiw2ujRo/VZLBafxWLxBwwYYFRcXExQfFZubi6xX79+RgwGw5HJZDoMHDjQsKysrME1zUGlUmV8Pr/q9u3bcl1bIpHA/fv3Gf7+/pV2dnbViYmJ8nMJCQkMiUSCw6LZTp06perj42PK5XLtMdmwZMkSrcYLvM2N5cePH1fDfquEhASV/v37GzEYDEdNTU37sWPH6lVXV8t1EkzXV/ytBg0aZKilpWWvzG915swZppWVlTWFQnHS19e33bhxo0bj+QQC8aNTUVFB1NTUbN
```

Comment:
For the title, I suggest:

- removing the seconds (especially if they are always 00)
- adding the timezone (guessing it's in UTC)

Reply:
I believe UTC is the timezone too...  Was able to update the title of the figure:
![image](https://user-images.githubusercontent.com/21114460/165635578-6839f2a3-97f5-46ed-b125-863bc34ad95e.png)

Reply:
I also updated the preview image to have the "new" title